Nos/Views/MuteCard.swift

```diff
@@ -3,14 +3,11 @@ import SwiftUI
 /// This view displays the information we have for an message suitable for being used in a list or grid.
 ///
 /// Use this view inside MessageButton to have nice borders.
-struct FollowCard: View {
+struct MuteCard: View {
```

Comment:
I'm a bit unsure about this name change. I get that it’s meant to limit the information and is intended for the mute list, but I’d love to see a bit more transparency. Maybe we could rename it to "ShortAuthorCard" so clients can use it whenever they want, or we could add a note in the comments explaining its purpose for the mute lists. I’m leaning towards the first option, but I’d really like to hear your thoughts on this.

Reply:
I'm torn about this. It's only used for the mute list, so naming it generically at this point feels like wishful thinking. On the other hand, I see the value in having the name be a little more descriptive.

I definitely need to update the doc comment, so I'll think about names while I do that.

Reply:
Yeah, I'm fine with updating the doc comment as well. The question I want to be answered there is, why it is called MuteCard if there is no code around muting inside or related to muting. Something like, a card that displays little information so that it can be safely used for displaying a muted author will do.

Reply:
Decided to update the name as well as the doc comment. Ready for another review.